app/services/embeddable_cache_refresh_service.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+class EmbeddableCacheRefreshService
+  EMBEDDABLE_CACHE_REFRESH_URL = "https://api.us.embeddable.com/api/v1/caching/refresh-contexts".freeze
```

Comment:
Small thing here:
This `.freeze` here is redundant because the file is already frozen.
Adding an explicit .freeze along with # frozen_string_literal: true is useful when we have an interpolated string (for example, "foo#{bar}") or an object (I think), like arrays, etc.
Not an issue though, Ruby won’t complain, but it doesn’t add anything.